tuf/pyca_crypto_keys.py

```diff
@@ -428,7 +428,7 @@ def verify_rsa_signature(signature, signature_method, public_key, data):
                                   salt_length=hashes.SHA256().digest_size), 
                                   hashes.SHA256())
 
-      verifier.update(data.encode('utf-8'))
+      verifier.update(data)
```

Comment:
The corresponding functions for `pycrypto_keys.py`will also need this change.

https://github.com/theupdateframework/tuf/blob/develop/tuf/pycrypto_keys.py#L297
https://github.com/theupdateframework/tuf/blob/develop/tuf/pycrypto_keys.py#L399

You can verify the required changes by running `test_pycrypto_keys.py`.
See the Travis build failure here: https://travis-ci.org/theupdateframework/tuf#L1294